server/server/api/views.py

```diff
@@ -241,194 +224,80 @@ def get(self, submenu_id=None):
             MenuItem:
                 type: object
                 properties:
-                    name:
-                        type: string
-                    num:
-                        type: number
                     uid:
-                        required: false
                         type: string
-                    lang_iso:
+                    root_name:
                         type: string
-                    lang_name:
+                    translated_name:
                         type: string
-                    id:
-                        required: false
+                    node_type:
                         type: string
+                    blurb:
+                        type: string
+                    acronym:
+                        type: string
+                    root_lang_iso:
+                        type: string
+                    root_lang_name:
+                        type: string
+                    child_range:
+                        type: string
+                    yellow_brick_road:
+                        type: boolean
                     children:
                         type: array
                         items:
                             type: MenuItem
-                    has_children:
-                        required: false
-                        type: boolean
-                    type:
-                        type: string
         """
         language = request.args.get(
             'language', current_app.config.get('DEFAULT_LANGUAGE')
         )
         return self.get_data(submenu_id, language=language), 200
 
-    def get_data(
-        self,
-        submenu_id=None,
-        menu_query=MENU,
-        submenu_query=SUBMENU,
-        language=None,
-        bind_vars=None,
-    ):
+    def get_data(self, submenu_id: str = None, language: str = None) -> List[dict]:
```

Comment:
how long does it take to generate a menu with empty cache?

Reply:
If we try to get the full menu (like `/api/menu`) the first time (or without cache) it can take almost 2-3 seconds and loads instantly with cache
Same with submenu like `/api/menu/{submenuId}`
The first time takes < 1 sec, and is almost instantaneous with cache